docs/faq.rst

```diff
@@ -45,6 +45,23 @@ repository is only modified from one place. Also keep in mind that
 Borg will keep an exclusive lock on the repository while creating
 or deleting archives, which may make *simultaneous* backups fail.
 
+Can I back up to multiple, swapped backup targets?
+--------------------------------------------------
+
+To protect my backups against data loss by malware or catastrophes like a fire I
+want to back up to weekly swapped-out USB disks. Then I place the media
+currently not in use offline and off-site to keep them safe. The swapping out
+can be as simple as switching between two disks regularly, or in more elaborate
+schemes like
+`Grandfather-Father-Son <https://en.wikipedia.org/wiki/Backup_rotation_scheme>`_.
+
+It is possible to swap your backup disks if each backup medium/set is assigned
+its own repository by creating a new one with :ref:`borg_init`. Borg needs to
+keep a local cache containing checksums of all file chunks already stored in the
+repositories. These caches are stored in subdirectories of ``~/.cache/borg/``.
+If Borg detects that a repository has been modified since the local cache was
+updated, it will rebuild the cache which can be quite time consuming.
```

Comment:
hmm, do we need to talk about cache / syncing in this context?

Reply:
Caching seems to be the main reason, so yes.

Reply:
Of course we could simply answer this faq with "Create one repo per media.", without any explaination.

Reply:
The latter was what I was thinking. :-)

That "if borg detects ..." is a general behaviour of borg and not related to swapping disks, but to updating the repo from another client.